src/manifolds/Sphere.jl

```diff
@@ -26,6 +26,8 @@ struct Sphere{N} <: AbstractEmbeddedManifold{DefaultIsometricEmbeddingType} end
 
 Sphere(n::Int) = Sphere{n}()
 
+base_manifold(M::Sphere) = M
```

Comment:
Why do we have this?

Reply:
That ends the (otherwise infinite) loop of locking for base manifolds... Oh, maybe here this line is not necessary because it has no `Val{N}`.

Reply:
This doesn't seem to be needed.

Reply:
You might be right. Let's remove it.

Reply:
There is already one definition for all `AbstractEmbeddedManifold`s so that's why it should work.